bam_consensus.c

```diff
@@ -1472,6 +1474,7 @@ int main_consensus(int argc, char **argv) {
         {"adj-MQ",             no_argument,       NULL, 'm'+100},
         {"no-adj-MQ",          no_argument,       NULL, 'm'+101},
         {"NM-halo",            required_argument, NULL, 'h'+100},
+        {"SC-cost",            required_argument, NULL, 'h'+101},
```

Comment:
I don't think there are any at the moment, but this way of selecting the value returned by long opts risks collisions...

Reply:
Ironically I started doing this when I did accidentally produce a clash when just using numerics.

I like to keep the options grouped in the list by function, but the numbers were being added sequentially as I added and extended the algorithm, which means you have to scan around to look for other numbers.

Note though that the compiler spots such errors (duplicate case statements) so it's very low risk